tests/jinja2/09-custom-jinja2-filters/suite.rc

```diff
@@ -1,28 +1,18 @@
 #!Jinja2
 
-{% set START_CYCLE = '01231212T1212' %}
-
 [scheduling]
-    initial cycle point = {{START_CYCLE}}
+    initial cycle point = 01231212T1212
     [[dependencies]]
         [[[R1]]]
             graph = strftime => pad => end
 [runtime]
     [[strftime]]
-        script = """
-if [[ "$HOURS:$MINUTES:$SECONDS" != '12:12:0' ]]; then
-    exit 1
-fi
-"""
+        pre-script = if [[ $TEST_1 != '00' ]]; then fail; fi
+        script = if [[ $TEST_2 != '30' ]]; then fail; fi
```

Comment:
`fail` is not a normal Unix command.

All you need are simple commands such as:
```sh
test "${TEST_1}" == '00'
```

`test` returns non-zero code if `"${TEST_1}"` is not `'00'`, which should trigger the error trap.